app/src/pages/serverErrorPage/ServerErrorPage.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import ServerErrorPage from './ServerErrorPage';
+import { act } from 'react-dom/test-utils';
+import userEvent from '@testing-library/user-event';
+
+const mockedUseNavigate = jest.fn();
+
+jest.mock('react-router', () => ({
+    useNavigate: () => mockedUseNavigate,
+    useLocation: () => jest.fn(),
+}));
+
+describe('ServerErrorPage', () => {
+    beforeEach(() => {
+        process.env.REACT_APP_ENVIRONMENT = 'jest';
+    });
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe('Rendering', () => {
+        it('renders page content with default error message and id', () => {
```

Comment:
renders page content with default error message and id when there is no error code